### DISCUSSION TITLE
External Data Items query failing with ResolveManagedPropertiesPlugin error

### DISCUSSION THREAD

**Discussion by brianpmccullough** [microsoft-search/pnp-modern-search#3183] [Q&A]
Added Graph Connector and attempting to use PnP Modern Search Results web part for display, but receiving the following error from the Search query api call.  There are many nested errors in the response, almost like a stack trace, but this is the one that I think was most "telling" to help determine what might be happening.

ResolveManagedPropertiesPlugin 
![20230807_154610](https://github.com/microsoft-search/pnp-modern-search/assets/8305245/cfb7252a-83b2-4f62-ae22-c8d70477778a)

I've confirmed I can create a custom result layout and vertical at the site scope, so it's not an issue of has the external data been setup correctly in search.

Within the web part, I've configured it to return only 2 of the many managed properties that I know to be there, and ensured it only asks for external items.  The query looks just like I would put into Graph Explorer to get results (haven't actually confirmed with Graph Explorer yet, as in this tenant, that permission hasn't been granted to Graph Explorer).  I have confirmed the ExternalItems.Read.All scope is consented to the SPFx App Registration (SharePoint Online Client Extensibility ...)

**Comment by wobba** [comment]
If you issue the same Graph search request using the Graph Explorer, do you get the same error? And seems from the error that the properties requested may be the issue. So looking at the JSON request could help. And verifying the properties you request are indeed in the connector schema.

**Comment by gretchunkim** [comment]
@brianpmccullough Are you still having this issue? There was a service degradation 365 Service Alert from Microsoft about external search results starting from 8/1/2023 to 8/10/2023. The issue supposed to be resolved, and I don't see any issue today.
